src/styles/_groundcontrol.scss

```diff
@@ -85,6 +85,28 @@
     .publish-voting {
       margin-top: 5px;
     }
+
+    .reassign-voting-warning {
+      font-size: 11px;
+      margin-left: 38px;
+    }
+
+    .confirm-assign-voting,
+    .current-voting-status {
+      max-height: 30px;
+      display: flex;
+      align-items: center;
+      gap: 6px;
+      margin-left: 15px;
+
+      .confirm-assign-text {
+        margin-top: 9px;
+      }
+
+      .voting-status-text {
+        margin-top: 10px;
+      }
+    }
```

Comment:
On the same vein as the above comment, these classes can be separated into a CSS module with the abstracted component name.